src/e2e-test/java/io/cdap/plugin/utils/PubSubClient.java

```diff
@@ -36,6 +56,22 @@ public static Topic createTopic(String topicId) throws IOException {
     }
   }
 
+  // Create the subscription
```

Comment:
Add the relevant comment under this
 /** 
 * Describe the function in this line.
*/